tests/integration/it_employees_test.py

```diff
@@ -47,22 +56,76 @@ def test_timestamp_mixin_updated_on(db_session):
     assert row.updated_on > old_updated_on
 
 
-def create_employee():
-    """ Create new instance of Employee to reuse in other tests """
-    return Employee(
-        first_name="John", last_name="Smith",
-        username="john", phone_number="123", birth_date=datetime.utcnow(),
-        registration_date=datetime.utcnow(), account_status="A",
-        user_status="Working", email="test@test.com", password="bla",
-        pin_code=1234
-    )
+def create_company():
+    """ Create new instance of Company to reuse it in other tests """
+    return Company(id=1,
+                   name="Acme Inc.",
+                   code="code1",
+                   address="addr"
+                   )
+
+
+def create_employee(company_id):
```

Comment:
@VladimirRyzhkov and this one will help you
https://github.com/timelesslounge/timelessis/pull/395/files#diff-882d8d4ba8a04d60cd877077a65f60f7R82